.envrc

```
@@ -0,0 +1 @@
+use flake
```

lets make it so that it only works with nix direnv installs
```suggestion
if has nix_direnv_version; then
  use flake
fi
```